wordcloud_style.py

```diff
@@ -29,24 +43,24 @@ def select_colour(self, word, value, word_to_values):
         #         return f"rgb({red},{green},{blue})"
         #     i += 1
```

Comment:
Consider this issue after solving the above...
What happens if the for loop ends but `return f"rgb({red},{green},{blue})"` is never hit? What does the function return in that case? Interestingly, the colour of None doesn't cause an error, but is invisible. This isn't right, so this case needs to be handled.